### PR TITLE
Otterspace badges [otterspace-badges]

### PR DESCRIPTION
**Pull request by rahulrumalla**
Fixes # .

Changes proposed in this pull request:
- Works with Goerli
- Adding a new strategy for using Otterspace badges (based on ERC4973, non-transferable tokens)

Will follow up with Optimism network once all end-to-end testing on goerli is verified. 

UPDATE
--------
- Added support for optimism as well

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hey @rahulrumalla 👋 Looks like some tests are failing (not returning any positive voting power) can you check by running this strategy locally? try running: `yarn test --strategy=otterspace-badges`

**Comment by rahulrumalla** [comment]
Ah! Had a change upstream in subgraph. Fixed and resolve merge conflict as well.

**Review by ChaituVR — APPROVED** [review]

File: src/strategies/otterspace-badges/index.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { error } from 'console';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'otterspace';
```
Comment (ChaituVR) [inline src/strategies/otterspace-badges/index.ts:4]:
```suggestion
export const author = 'otterspace-xyz';
```

Comment (ChaituVR) [reply src/strategies/otterspace-badges/index.ts:4]: I forgot about this @rahulrumalla should be your GitHub id or org's GitHub id

**Review by ChaituVR — CHANGES_REQUESTED** [review]
GitHub id missing

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**